src/WrapperApp/components/Simulation/SimulationCard.tsx

```diff
@@ -97,6 +97,8 @@ export default function SimulationCard({
 				return 'success.main';
 			case StatusState.LOCAL:
 				return 'warning.main';
+			case StatusState.CANCELED:
```

Comment:
rename CANCELED to CANCELLED

Reply:
I can change the enum label but the string value has to be changed on the backend as well for everything to work.

Reply:
OK, then lets keep it